runtime/onert/backend/train/MemoryPlanner.h

```diff
@@ -147,10 +156,10 @@ class WICPlanner : public basic::IMemoryPlanner<DisposableTensorIndex>
   bool _initialized;
   uint32_t _capacity;
   MemoryPlans _mem_plans;
-  std::unordered_set<DisposableTensorIndex> _live_indices;
-  DisposableTensorIndexMap<std::vector<DisposableTensorIndex>> _interference_graph;
+  std::unordered_set<Index> _live_indices;
+  std::unordered_map<Index, std::vector<Index>> _interference_graph;
```

Comment:
This PR updates `DisposableTensorIndexMap<std::vector<DisposableTensorIndex>>` to `std::unordered_map<Index, std::vector<Index>>`.  

If so, `DisposableTensorIndexMap` is not used anymore. So, delete here ->